### DISCUSSION TITLE
Remix Jokes Tutorial // All links to Remix documentation are failing

### DISCUSSION THREAD

**Discussion by fre-ben** [remix-run/remix#750] [Q&A] [answered]
Hey everyone!
It seems that every link inside the Remix Jokes Tutorial (https://remix.run/docs/en/v1/tutorials/jokes) is linking to non-existing pages.

Example: https://remix.run/docs/en/v1/tutorials/jokes#read-from-the-database-in-a-remix-loader
`loader` component is linked to https://remix.run/docs/en/v1.0.5/api/conventions#loader, which results in a 404.

Is this already being taken care of in some PR? Couldn't really find anything based on a simple search.

**Comment by IshanKBG** [comment, marked as answer]
I clicked on the loader link in the tutorial and it redirects me to the right link
Maybe can you try again ?
